hack/dind-run-operators.sh

```diff
@@ -1,7 +1,8 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
+helm init --upgrade
 ./manifests/local-dind/setup.sh
 kubectl apply -f ./manifests/crd.yaml
-helm install charts/tidb-operator --name tidb-operator --namespace=tidb-operator
+helm install charts/tidb-operator --name tidb-operator --namespace=tidb-admin --set "imagePullPolicy=Always"
```

Comment:
If we use this script in the tutorial, we should keep the namespace of tidb-operator consistent.

Reply:
BTW, we plan to modify dind-cluster.sh to adjust for our test, because we can hardly deploy a docker registry for development with the upstream dind-cluster.sh. `kubectl port-forward` works for Linux but not Mac.

Reply:
Do you like the name "tidb-admin"? With "operator", it only makes sense to run the operator (which is what we are doing at the moment). With "tidb-admin" it would make sense to run additional utilities under that name. I think it captures everything a user needs to know: "tidb": its for tidb. "admin", its for administrative duties for tidb.

I have seen port-forward working fine on Mac. But it doesn't keep up with changes properly?

Reply:
I'm OK with tidb-admin. kubectl port-forward works for normal service but not for docker registry, since Docker for Mac runs Docker daemon inside a VM and port-forward can only bind localhost. We can't push Docker images through Docker daemon to VM's localhost.